app/src/test/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModelTest.java

```diff
@@ -0,0 +1,96 @@
+package com.kickstarter.viewmodels;
+
+import android.util.Pair;
+
+import com.kickstarter.KSRobolectricTestCase;
+import com.kickstarter.factories.PhotoFactory;
+import com.kickstarter.factories.ProjectFactory;
+import com.kickstarter.models.Project;
+
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import rx.observers.TestSubscriber;
+
+public final class ProjectSearchResultHolderViewModelTest extends KSRobolectricTestCase {
+  private ProjectSearchResultHolderViewModel.ViewModel vm;
+  private final TestSubscriber<Project> notifyDelegateOfResultClick = new TestSubscriber<>();
+  private final TestSubscriber<String> projectImage = new TestSubscriber<>();
+  private final TestSubscriber<String> projectName = new TestSubscriber<>();
+  private final TestSubscriber<Pair<Integer, Integer>> projectStats = new TestSubscriber<>();
+
+  @Before
+  public void setUpEnvironment() {
+    this.vm = new ProjectSearchResultHolderViewModel.ViewModel(environment());
```

Comment:
i know we have access to the environment within our test files, but it's good practice to pass in the environment in the method in case we need to modify it within each test in the future